include/device.h

```diff
@@ -757,10 +757,19 @@ BUILD_ASSERT(sizeof(device_handle_t) == 2, "fix the linker scripts");
 		(&DEVICE_NAME_GET(dev_name)), level, prio)
 
 #ifdef CONFIG_PM_DEVICE
-#define Z_DEVICE_DEFINE_PM_INIT(dev_name, pm_control_fn)		\
+#define Z_DEVICE_STATE_DEFINE(node_id, dev_name)			\
+	static struct device_state Z_DEVICE_STATE_NAME(dev_name) = {	\
+		.pm = {							\
+			.atomic_flags = DT_PROP_OR(node_id, wakeup_source, 0), \
```

Comment:
I think this can be directly initialized:
```c
.atomic_flags = ATOMIC_INIT((DT_PROP_OR(node_id, wakeup_source, 0) << PM_DEVICE_ATOMIC_FLAGS_WS_CAPABLE_BIT))
```

Reply:
nice ! Will do it, was not worried about the value because this is properly set when the device is initialized, but have the proper value in the definition is more accurate.